templates/tempest_skip_list.yml.j2

```diff
@@ -1,4 +1,4 @@
 ---
-{% for key, value in tempest_skip_tests.items() %}
+{% for key, value in item.value.tempest_skip_tests.items() %}
```

Comment:
After changing this, and users of this role will have to update their {{ clouds }} variable (with the tempest_skip_lists) to keep the role working. This is fine with me as long as it's documented.

Reply:
ok, will do